src/Selection.js

```diff
@@ -1,12 +1,15 @@
 import contains from 'dom-helpers/query/contains'
 import closest from 'dom-helpers/query/closest'
-import events from 'dom-helpers/events'
 
 function addEventListener(type, handler, target = document) {
-  events.on(target, type, handler, { passive: false })
+  target.addEventListener(type, handler, { capture: true, passive: false })
```

Comment:
I know you worked on this on another task.. but to not forget: we need to work a bit on how we pass events in RBC and React17
Unfortunately, this change makes huge problem for many therapists, see video:

https://healthadvisorch.slack.com/files/U5JCTV5C2/F03UREGM9MK/agenda-accidental-move.mp4
see thread: https://healthadvisorch.slack.com/archives/C5JCSBGNN/p1661279058436689

in the end, on prod I've dropped changes related to moving event fixes, so you are able to open the event without having it in the dragging state :wink: